scripts/get_resource_names.go

```diff
@@ -0,0 +1,229 @@
+// get_resource_names collects the resource names from the test files in test-artifacts
+// By default, it skips the names starting with 'Test' or 'test', as they are otherwise
+// recognised and handled during the leftover removal operation.
+// The resulting list has to be added to `remove_leftover_test.go`
```

Comment:
The file is `...leftovers...go` (plural).

Reply:
fixed